app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -527,17 +527,28 @@ class MigrationsProvider(val context: Context) {
         }
     }
 
-    // todo: This is VPN project migration, KEEP IT ALWAYS LAST
     val MIGRATION_40_TO_41: Migration = object : Migration(40, 41) {
         override fun migrate(database: SupportSQLiteDatabase) {
             database.execSQL(
                 "CREATE TABLE IF NOT EXISTS `web_trackers_blocked` (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, " +
                     "trackerUrl TEXT NOT NULL, trackerCompany TEXT NOT NULL, timestamp TEXT NOT NULL)"
             )
-            // place here new migrations from the main app
-            // this avoids crashes and forcing to uninstall the appTP app every time a new migration
-            // is added to the main app
-            MIGRATION_39_TO_40.migrate(database)
+        }
+    }
+
+    @Suppress("DEPRECATION")
+    val MIGRATION_41_TO_42: Migration = object : Migration(41, 42) {
```

Comment:
Added migration and tested by upgrading from 5.101.0 (where the first experiment was added) to latest. Tested new / returning scenarios and the DB was updated only for returning users with the established status.